plugin/src/main/java/org/opensearch/ml/model/MLModelGroupManager.java

```diff
@@ -77,7 +77,7 @@ public void createModelGroup(MLRegisterModelGroupInput input, ActionListener<Str
                                     new IllegalArgumentException(
                                         "The name you provided is already being used by another model with ID: "
                                             + id
-                                            + ". Please provide a different name"
+                                            + ". Please provide a different name or add \"model_group_id\": \"lMPmr4kB4eSCtCCDmCDm\" to request body"
```

Comment:
Should we update to `Please provide a different group name`. We can see people started having confusion: https://github.com/opensearch-project/ml-commons/issues/1179

about: `\"model_group_id\": \"lMPmr4kB4eSCtCCDmCDm\"` :  Is this any static group id we are adding?

Reply:
Fixed hard coded model id .
For `Please provide a different model group name` , that looks also confusing as user just provide model name in the request body, not model group name,

```
POST _plugins/_ml/models/_upload
{
  "name": "huggingface/sentence-transformers/all-MiniLM-L12-v2",
  "version": "1.0.1",
  "model_format": "TORCH_SCRIPT"
}
```